MembraneRTC/src/main/java/org/membraneframework/rtc/InternalMembraneRTC.kt

```diff
@@ -141,10 +141,7 @@ constructor(
         }
 
         localTracks.add(screencastTrack)
-        localPeer = localPeer.withTrack(screencastTrack.id(), metadata + mapOf(
-            "type" to "screensharing",
-            "user_id" to (localPeer.metadata["displayName"] ?: "")
-        ))
+        localPeer = localPeer.withTrack(screencastTrack.id(), metadata)
```

Comment:
I'd do it like this: 
```suggestion
        localPeer = localPeer.withTrack(screencastTrack.id(), mapOf(
             "user_id" to (localPeer.metadata["displayName"] ?: "")
        ) + metadata)
```
that way user can overwrite user_id to whatever they want